common/kvstore/src/main/java/org/apache/spark/util/kvstore/InMemoryStore.java

```diff
@@ -79,55 +74,49 @@ public long count(Class<?> type, String index, Object indexedValue) throws Excep
 
   @Override
   public <T> T read(Class<T> klass, Object naturalKey) {
```

Comment:
Is `Class<T> klass` vs. `Class<?> type` an intentional choice or an accidental consistency? Either way, I'm not sure that I am following the logic of the convention.

Reply:
Hmmm... also not entirely consistent across related files.

Reply:
I did notice this, but I didn't follow up and look.  Thinking about it before looking, I assume Class<T> type doesn't work in scala, while Class[T] klass would.  Let me see what's with that, though....

Reply:
I was thinking more about naming convention, `klass` vs. `type` -- more about understandability than functionality.

Reply:
I think it's 'type' because KVTypeInfo.
But KVStore seems to be mixed, and ElementTrackingStore uses klass always.